sql/branch.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
+    UPDATE payment_instructions SET due = floor(9.41/(amount)) * amount WHERE due > 9.41;
```

Comment:
Alright, so what's up with these updates. Why are these the right repairs for our situation?